src/components/select/Select.tsx

```diff
@@ -466,6 +467,7 @@ function Select<Option extends OptionType, IsMulti extends boolean, IsClearable
           ClearIndicator,
           DropdownIndicator,
           IndicatorSeparator: null,
+          Input: (inputProps) => <ReactSelectComponents.Input data-is-select="true" {...inputProps} />,
```

Comment:
B?: Components should be defined outside of the render function, this will cause the input to rerender more than neccesary.
https://react-select.com/components#defining-components